src/es5processor.ts

```diff
@@ -220,16 +221,24 @@ class ES5Processor extends Rewriter {
         //   __export(x);
         // This extra variable is necessary in case there's a later import of the
         // same module name.
-        require = this.isExportRequire(call);
-        isExport = require != null;
+        const innerCall = this.isExportRequire(call);
+        if (!innerCall) return false;
+        isExport = true;
+        call = innerCall;  // Update call to point at the require() expression.
+        require = this.isRequire(call);
       }
       if (!require) return false;
 
       this.writeLeadingTrivia(node);
       const varName = this.emitGoogRequire(null, require);
 
       if (isExport) {
-        this.emit(`__export(${varName});`);
+        // node is a statement containing a require() in it, while
+        // requireCall is that call.  We replace the require() call
+        // with the variable we emitted.
+        let fullStatement = node.getText();
```

Comment:
maybe we should lint tsickle like in google3, ought to be const

but that would be a separate task to change it throughout

Reply:
Done.